gradle.properties

```diff
@@ -1,15 +1,16 @@
 pluginGroup=org.ladybird.dsl
 pluginName=Ladybird-DSL
 pluginVersion=2.0.2
```

Comment:
I am far from an expert here, but this seems like a change that would be worth bumping the version number and adding to the changelog.

Reply:
definitely, just based on the (not many) previous PRs, it looks like Andrew usually bumps the version number directly in master, and not as part of the PR. Would be happy to do it myself though!